pynopegl-utils/pynopegl_utils/tests/debug.py

```diff
@@ -26,6 +26,23 @@
 import pynopegl as ngl
 
 
+def get_grid_coordinates(cols: int, rows: int) -> Mapping[str, Tuple[float, float]]:
```

Comment:
Can we somehow align the names of the two functions ? `get_grid_points()` maybe ?

Reply:
They don't return the same type of data; one is a data dict and the other is a node tree; I'd rather not imply they're the same type of function

Reply:
But that is given by the functions signatures. `get_grid_coordinates()` is used like this `points=get_grid_coordinates(_N, _N)` so why not call it `get_grid_points()` ?

Reply:
I would also make sense to put that function in `cmp_cuepoints.py`.

Reply:
I have conflicting thoughts about all of this:
- "that is given by the functions signatures", yeah but well "get_{grid,debug}_points()" being totally different function types is misleading
- `get_debug_points()` might be misnamed, so maybe a change of name is required first
- having functions in `cmp_cuepoints.py` is weird because the `cmp_*` now only expose a single class/function, and it feels like hacking random utils into them; it would also imply moving `get_debug_points()` there because it's only used in the context of cuepoints

Reply:
Submitted a new proposition.